proposals/wgs/wg-charter.md

```diff
@@ -0,0 +1,71 @@
+# WG Charter Guide
+
+All Kubeflow WGs must define a charter defining the scope and governance of the WG.
+
+- The scope must define what areas the WG is responsible for directing and maintaining.
+- The governance must outline the responsibilities within the WG as well as the roles
+  owning those responsibilities.
+
+## Steps to create a WG charter
+
+1. Copy [the template][Short Template] into a new file under community/wg-*YOURWG*/charter.md ([sig-architecture example])
+2. Read the [Recommendations and requirements] so you have context for the template
+3. Fill out the template for your WG
+4. Update [wgs.yaml] with the individuals holding the roles as defined in the template.
+5. Add subprojects owned by your SIG in the [wgs.yaml]
+5. Create a pull request with a draft of your charter.md and wgs.yaml changes.  Communicate it within your WG
+   and get feedback as needed.
+6. Send the WG Charter out for review to steering@kubeflow.org.  Include the subject "WG Charter Proposal: YOURWG"
+   and a link to the PR in the body.   
+7. Typically expect feedback within a week of sending your draft.  Expect longer time if it falls over an
+   event such as KubeCon/CloudNativeCon or holidays.  Make any necessary changes.
+8. Once accepted, the steering committee will ratify the PR by merging it.
+
+## Steps to update an existing WG charter
+
+- For significant changes, or any changes that could impact other WGs, such as the scope, create a
+  PR and send it to the steering committee for review with the subject: "WG Charter Update: YOURWG"
```

Comment:
Should the email be also mentioned here? (Or is it jjust a PR?)

Reply:
Maybe in the future. I don't think the email group is setup yet so I hestitate to mention it here yet.